server/src/main/scala/org/broadinstitute/dsp/workbench/welder/server/Main.scala

```diff
@@ -17,27 +18,28 @@ import org.broadinstitute.dsp.workbench.welder.JsonCodec._
 import org.http4s.server.blaze.BlazeServerBuilder
 
 import scala.concurrent.ExecutionContext
-import scala.concurrent.duration._
 
 object Main extends IOApp {
   override def run(args: List[String]): IO[ExitCode] = {
     implicit val unsafeLogger = Slf4jLogger.getLogger[IO]
 
     val app: Stream[IO, Unit] = for {
+      blockingEc <- Stream.resource[IO, ExecutionContext](ExecutionContexts.fixedThreadPool(255))
+      implicit0(it: Linebacker[IO])  = Linebacker.fromExecutionContext[IO](blockingEc)
       appConfig <- Stream.fromEither[IO](Config.appConfig)
```

Comment:
Does this handle the case I hit today, where `appConfig` returned an error?

Reply:
it's not this change that addresses the issue. Reason why the app hangs previously is https://github.com/DataBiosphere/welder/pull/13/files#diff-5b4ceaa24c625c1442c692ab05b7b26dR41..So previous, we recover from any error that happens during initialization, and log the error, but then never throw the exception. Hence, it reaches `IO.never` as usual, which hangs the app.

The change will throw any exception during initialization instead.